.github/workflows/build-triton-wheel.yml

```
@@ -62,7 +62,7 @@ jobs:
       DOCKER_IMAGE: ${{ matrix.device == 'rocm' && format('pytorch/manylinux2_28-builder:rocm{0}', matrix.rocm_version) || matrix.docker-image }}
       PY_VERS: ${{ matrix.py_vers }}
       BUILD_DEVICE: ${{ matrix.device }}
-      PLATFORM: ${{ contains(matrix.docker-image, '2_28') && 'manylinux_2_28_x86_64' || 'manylinux2014_x86_64' }}
+      PLATFORM: ${{ contains(matrix.docker-image, '2_28') && 'manylinux_2_28_x86_64' }}
```

Ops, this turns `PLATFORM` into a boolean and fails the build
```suggestion
      PLATFORM: 'manylinux_2_28_x86_64'
```

---

I think `pytorch/manylinux-builder:cpu` on line 49 would also need to be removed